src/libtiled/tilelayer.h

```diff
@@ -371,6 +421,11 @@ inline bool TileLayer::contains(const QPoint &point) const
     return contains(point.x(), point.y());
 }
 
+inline QPair<int, int> TileLayer::block(int x, int y) const
```

Comment:
I find the name of this function not really describing... it sounds like it `blocks` something... But it actually calculates the block coordinates...